runtime/parachains/src/configuration.rs

```diff
@@ -130,6 +130,8 @@ pub struct HostConfiguration<BlockNumber> {
 	///
 	/// This parameter affects the upper bound of size of `CandidateCommitments`.
 	pub hrmp_max_message_num_per_candidate: u32,
+	/// The amount of block weight which we assume is consumed to process a single backed candidate.
+	pub backed_candidate_block_weight: u32,
```

Comment:
Why is that a configuration value?

This value is calculated by benchmarking and not by the council or whatever.

Reply:
Will change it.

Reply:
174f9064